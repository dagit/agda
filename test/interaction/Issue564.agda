module Issue564 where

postulate
  Level : Set
  zero  : Level

{-# BUILTIN LEVEL     Level #-}
{-# BUILTIN LEVELZERO zero  #-}

postulate
  A : Level → Set

module M ℓ where
  postulate a : A ℓ

postulate
  P : A zero → Set

open M zero

p : P a
p = {!!}
