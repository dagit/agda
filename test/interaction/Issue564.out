Prelude> Agda2> (agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking Issue564 (Issue564.agda).\n" t)
(agda2-info-action "*Type-checking*" "Finished Issue564.\n" t)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : P a\n" nil)
((last . 1) . (agda2-goals-action '(0)))
Agda2> (agda2-status-action "")
(agda2-info-action "*Current Goal*" "P a" nil)
((last . 1) . (agda2-goals-action '(0)))
Agda2> 